index.html

```diff
@@ -1,49 +1,242 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
+  
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Week 1: HTML Tables</title><!-- Latest compiled and minified CSS -->
+  <title>National Homelessness</title>
+
   <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <link href='https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,700' rel='stylesheet' type='text/css'>
+  
   <style>
-    .header {
-      text-align: center;
+    .section {padding: 1em 0;} 
+
+    .header {text-align: left;} 
+
+    #data-table td {width: 25%;}
+
+    .bar {height: 20px; background: #F67280;}
+
+    h1 {
+      color:#355C7D;
+      font-family: 'Open Sans Condensed', Helvetica, sans-serif;
+      font-size: 300%;
+    } 
+
+    p {
+      color:#355C7D;
+      font-family: 'Open Sans Condensed', Helvetica, sans-serif;
+      font-size: 90%;
     }
-  </style>
+
+    table {
+      border-collapse: collapse;
+      width: 100%;
+      white-space: nowrap; 
+      font-family: 'Open Sans Condensed', Helevetica, sans-serif;
+      font-weight: 500;
+      color: #F67280;
+    }
+
+    th { 
+      background-color: #355C7D;
+      font-family: 'Open Sans Condensed', Helvetica, sans-serif;
+      color: white;
+    }
+    
+    tr:nth-child(even) {background-color:#6C5B7B}
+
+    tr:hover td{font-weight: bold;}
+
+    body {background-color: #F8B195;}
+   </style>
+
 </head>
+
 <body>
-  <div class="container">
-    <div class="header">
-      <h1>Week 1: HTML Tables</h1>
-      <p>We'll add our data to this table.</p>
+
+    <div class="header section">
+      <div class="container">
+        <h1>Homelessness in the United States</h1>
+        <p>Data Source: National Alliance to End Homelessness</p>
+      </div>
+    </div>
+    
+    <div class="controls section">
+      <div class="container">
+        <div class="btn-toolbar">
+          <div class="btn-group" role="group" aria-label="Sort">
+            <button type="button" class="btn btn-default" id="sort-asc">Ascending</button>
+            <button type="button" class="btn btn-default" id="sort-desc">Descending</button>
+          <div class="btn-group" role="group" aria-label="Filter">
+            <button type="button" class="btn btn-default" id="filter-byrate">Filter by Rate > 15%</button>
+          </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="main section">
+      <div class="container">
+        <table class="table" id="data-table">
+          <thead>
+            <tr>
+              <th>State</th>
+              <th>2014 Rate per 10K</th>
+              <th>Rate by Comparison</th>
```

Comment:
I didn't know what this meant. Of course, not a big deal (I called it "chart," didn't I?), just flagging that in this case, maybe no label would be better than a confusing label.